difftest/dpi_t1rocketemu/src/dpi.rs

```diff
@@ -374,11 +277,20 @@ unsafe extern "C" fn t1_cosim_init(
 
   let scope = SvScope::get_current().expect("failed to get scope in t1_cosim_init");
 
+  use std::io::Write;
+  let ds3_cfg = include_bytes!("dramsim3-config.ini");
+  let mut ds3_cfg_file =
+    tempfile::NamedTempFile::new().expect("Unable to create DRAMsim3 configuration temp file");
+  ds3_cfg_file.write(ds3_cfg).expect("Unable to write DRAMsim3 configuration temp file");
+  let (_, ds3_cfg_path) =
+    ds3_cfg_file.keep().expect("Unable to persist DRAMsim3 configuration temp file");
+
   let args = OnlineArgs {
     elf_file: elf_file.get().to_str().unwrap().into(),
     dlen: dlen as u32,
     vlen: vlen as u32,
     spike_isa: spike_isa.get().to_str().unwrap().into(),
+    dramsim3_cfg: Some(ds3_cfg_path),
```

Comment:
I this here should not be a optionals dramsim will always linking.

Reply:
I think using the trivial model is desirable in some cases?